codacy/requirements.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
     global.metricsdb.create, global.filestoredb.create, global.jobsdb.create
 
 - name: log-router
-  version: 0.3.0
+  version: 0.4.1
```

Comment:
This will also need to change on the requirements-dev.yaml file.

Reply:
unrelated, but related. Your fix for namespaces was never deployed since you never updated to 0.4.0.

Reply:
Yep I just found that out too.  :)